avm/res/net-app/net-app-account/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,48 @@
+targetScope = 'subscription'
+
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+// e.g., for a module 'network/private-endpoint' you could use 'dep-dev-network.privateendpoints-${serviceShort}-rg'
+param resourceGroupName string = 'dep-${namePrefix}-netapp.netappaccounts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = 'westus3'
```

Comment:
Will be overwritten. Please refer to https://github.com/Azure/bicep-registry-modules/pull/792#discussion_r1455072261

Reply:
adjusted with tempLocation